nnpdf_data/nnpdf_data/commondata/HERA_CC_318GEV/metadata.yaml

```diff
@@ -123,5 +126,8 @@ implemented_observables:
     legacy:
       data_uncertainties:
       - uncertainties_legacy_EP-SIGMARED.yaml
+    reimplemented:
+      data_uncertainties:
```

Comment:
Instead of a variant call "reimplemented" this should be the standard dataset, so put this in data uncertainties.

Also, if the legacy and new uncertainties are numerically equal, you can remove the legacy uncertainties.yml file and make the legacy variant point to your new file. This will facilitate cleaning up legacy variants later.